App/Containers/Groups.tsx

```diff
@@ -242,7 +243,7 @@ class Groups extends React.PureComponent<Props, State> {
 }
 
 const mapStateToProps = (state: RootState): StateProps => {
-  const ownAddress = state.account.address.value
+  const ownAddress = getAddress(state)
```

Comment:
moving these to selectors, they might also require a non-null check in the chain... shouldn't but our crashes indicate something null happening that shouldn't be